.github/workflows/be_review_prs.yml

```diff
@@ -0,0 +1,52 @@
+name: Enforce PR Approval Requirements
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize, ready_for_review]
+
+jobs:
+  check-approval-requirements:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Get PR details
+        id: pr_details
+        uses: octokit/request-action@v2.x
+        with:
+          route: GET /repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/requested_reviewers
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Check required reviewers
+        id: check_reviewers
+        run: |
+          REQUESTED_TEAMS=$(echo '${{ steps.pr_details.outputs.data }}' | jq -r '.teams[].slug')
+          if echo "$REQUESTED_TEAMS" | grep -qiE '(mobile|lighthouse|identity)'; then
+            echo "Specialized team review required. Skipping backend-review-group requirement."
+            echo "skip_backend_review=true" >> $GITHUB_OUTPUT
+          else
+            echo "No specialized team review required. Enforcing backend-review-group requirement."
+            echo "skip_backend_review=false" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Check backend-review-group approval status
+        if: steps.check_reviewers.outputs.skip_backend_review == 'false'
+        id: check_backend_review_group_approval_status
+        uses: octokit/request-action@v2.x
+        with:
+          route: GET /repos/${{ github.repository }}/pulls/${{ github.event.pull_request.number }}/reviews
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        
+      - name: Verify backend-review-group approval
+        if: steps.check_reviewers.outputs.skip_backend_review == 'false'
+        run: |
+          APPROVALS=$(echo '${{ steps.check_backend_review_group_approval_status.outputs.data }}' | jq '[.[] | select(.state == "APPROVED" and .author_association == "MEMBER" and (.user.login == "backend-review-group" or .user.login | contains("backend-review-group")))] | length')
+          if [ "$APPROVALS" -eq 0 ]; then
+            echo "Error: This PR requires approval from backend-review-group."
+            exit 1
+          else
+            echo "Backend-review-group approval confirmed."
+          fi
```

Comment:
@rjohnson2011 Can we have this post a comment on the PR too? An example of where we create a PR comment [is here](https://github.com/department-of-veterans-affairs/vsp-infra-application-manifests/blob/main/.github/workflows/force-separate-prod-vs-lower-env-prs.yaml#L44-L49)